builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -191,7 +191,7 @@ end
 
 local function queue_download(package, reason)
 	local max_concurrent_downloads = tonumber(core.settings:get("contentdb_max_concurrent_downloads"))
-	if number_downloading < max_concurrent_downloads then
+	if number_downloading < math.max(max_concurrent_downloads, 1) then
```

Comment:
0 or 1 ?

Reply:
1. you want at least 1 download in the queue.